controller/api/models.py

```diff
@@ -179,6 +179,28 @@ def __str__(self):
     def url(self):
         return self.id + '.' + settings.DEIS_DOMAIN
 
+    def _get_job_id(self, container_type):
+        app = self.id
+        release = self.release_set.latest()
+        version = "v{}".format(release.version)
+        job_id = "{app}_{version}.{container_type}".format(**locals())
+        return job_id
+
+    def _get_command(self, container_type):
```

Comment:
I am not having the container object at the place where i am using it.I can use the container's function by passing the container's list from the scale and deploy to functions where i am using it and finding the unique types and count from the lists.